apps/api/gql/schema/pool.gql

```diff
@@ -629,6 +656,10 @@ interface GqlPoolBase {
     Hook assigned to a pool
     """
     hook: GqlHook
+    """
+    QuantAmmWeightedParams
+    """
+    quantAmmWeightedParams: QuantAmmWeightedParams
```

Comment:
also not needed in the base, should implement a new type instead which implements the base